spring-cloud-dataflow-server-core/pom.xml

```diff
@@ -306,6 +306,44 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<goals>
+							<goal>build-info</goal>
+						</goals>
+						<configuration>
+							<additionalProperties>
+								<encoding.source>UTF-8</encoding.source>
```

Comment:
What are these additional props needed for? These would be `build.encoding.source=UTF-8` and `build.encoding.reporting=UTF-8`.

Reply:
Not really, I'll remove them...
Done